### PR TITLE
gnomeExtensions.topicons-plus: 22 -> 27

### PR DESCRIPTION
**Pull request by eduardosm**
###### Motivation for this change

Unmark as broken because it is compatible with the current version of GNOME.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dotlambda** [comment]
It will be broken again soon: https://github.com/phocean/TopIcons-plus/issues/155.
See https://github.com/NixOS/nixpkgs/issues/116252.

**Comment by emanueleperuffo** [comment]
Isn't this extension unsupported and unmaintained?

https://github.com/phocean/TopIcons-plus/issues/91

**Comment by eduardosm** [comment]
> It will be broken again soon: [phocean/TopIcons-plus#155](https://github.com/phocean/TopIcons-plus/issues/155).
> See #116252.

> Isn't this extension unsupported and unmaintained?
> 
> [phocean/TopIcons-plus#91](https://github.com/phocean/TopIcons-plus/issues/91)

This version works with the current version of GNOME and there is a PR to make it compatible with GNOME 40.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118306` at d602c233 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>gnomeExtensions.topicons-plus</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `gnome3`.
Near pkgs/desktops/gnome-3/extensions/topicons-plus/default.nix:1:39:
```
  |
1 | { lib, stdenv, fetchFromGitHub, glib, gnome3, gettext }:
  |                                       ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, please check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/desktops/gnome-3/extensions/topicons-plus/default.nix:24:5:
```
   |
24 |     license = licenses.gpl2;
   |     ^
```
</li>
  </ul>
</details>

**Comment by eduardosm** [comment]
> 2 suggestions:

Fixed

File: pkgs/desktops/gnome-3/extensions/topicons-plus/default.nix [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
   pname = "gnome-shell-extension-topicons-plus";
-  version = "22";
+  version = "27";
 
   src = fetchFromGitHub {
     owner = "phocean";
     repo = "TopIcons-plus";
-    rev = "v${version}";
-    sha256 = "196s1gdir52gbc444pzrb5l7gn5xr5vqk5ajqaiqryqlmp3i8vil";
+    rev = "${version}";
```
Comment (SuperSandro2000) [inline pkgs/desktops/gnome-3/extensions/topicons-plus/default.nix:10]:
```suggestion
    rev = version;
```

Comment (eduardosm) [reply pkgs/desktops/gnome-3/extensions/topicons-plus/default.nix]: Squashed

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Pull request merged**